tests/unit/nupic/research/temporal_memory_test.py

```diff
@@ -999,6 +999,27 @@ def testMapCellsToColumns(self):
     self.assertEqual(columnsForCells[99], set([399]))
 
 
+  def testMaxSegmentsPerCellGetter(self):
+    tm = TemporalMemory(
+      columnDimensions=[64,64],
+      cellsPerColumn=32,
+      maxSegmentsPerCell=200
+    )
+    self.assertEqual(tm.connections.maxSegmentsPerCell, tm.getMaxSegmentsPerCell())
+    self.assertEqual(tm.connections.maxSegmentsPerCell, 200)
+    self.assertEqual(tm.getMaxSegmentsPerCell(), 200)
```

Comment:
Same comment as the others below. You only need this one assertion: 

    self.assertEqual(tm.getMaxSegmentsPerCell(), 200)